CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ set(CMAKE_CXX_FLAGS
 	-Werror=uninitialized
 	-Werror=init-self
 	-Wcast-qual
-	-Wconversion
+	-Wno-conversion
```

Comment:
We use the same compiler flags as px4 so not really appealing to change them. When you declare a constant, just put an f at the end and it will be a float constant instead of a double constant.